wikibase/1.30/bundle/LocalSettings.php.wikibase-bundle.template

```diff
@@ -3,3 +3,6 @@ wfLoadExtension( 'OAuth' );
 ${DOLLAR}wgGroupPermissions['sysop']['mwoauthproposeconsumer'] = true;
 ${DOLLAR}wgGroupPermissions['sysop']['mwoauthmanageconsumer'] = true;
 ${DOLLAR}wgGroupPermissions['sysop']['mwoauthviewprivate'] = true;
+
+# WikibaseImport
+require_once "${DOLLAR}IP/extensions/WikibaseImport/WikibaseImport.php";
```

Comment:
Any reason not to use `wfLoadExtension( 'WikibaseImport' )`?

Reply:
I just followed the README.md.

I'm happy to use wfLoadExtension though

Reply:
I tried but apparently it means the script doesn't work:
https://github.com/filbertkm/WikibaseImport/issues/32